consensus/ibft/hooks.go

```diff
@@ -18,6 +18,10 @@ const (
 	// PoS defines the Proof of Stake IBFT type,
 	// where the validator set it changed through staking on the Staking SC
 	PoS MechanismType = "PoS"
+
+	// Bridge defines Bridge type
+	// Bridge allows to include StateTransactions into block
```

Comment:
Typo, should be `Bridge allows the inclusion of StateTransactions into a block`

Reply:
@zivkovicmilos To be honest, I don't like IBFT knows what hooks each mechanism wants because the mechanism is an extension and IBFT must not know them.

Reply:
Fixed at 4a749c116ca9d4fda4904e06e5780dc40530e00f